src/sbt-test/sbt-guardrail/java-codegen-app/build.sbt

```diff
@@ -0,0 +1,28 @@
+
+enablePlugins(GuardrailPlugin)
+
+name := "sbt-guardrail-java-test-app"
+
+version := "1.0." + System.currentTimeMillis
+
+scalaVersion := "2.12.10"
+
+scalacOptions += "-Xexperimental"
+
+guardrailTasks in Compile := List(
+  JavaClient(file("petstore.json"), pkg="com.example.clients.petstore")
+)
+
+// workaround for SBT issue
```

Comment:
What's the issue? How do we know when this can be removed?

Reply:
the issue is https://github.com/sbt/sbt/issues/1664  (opened in October 2014)

Reply:
I updated my comment in build.sbt